CMakeLists.txt

```diff
@@ -1,23 +1,23 @@
-cmake_minimum_required(VERSION 3.7)
-
-if(POLICY CMP0091)
-	cmake_policy(SET CMP0091 NEW) # Support MSVC_RUNTIME_LIBRARY
+cmake_minimum_required(VERSION 3.10...3.20 FATAL_ERROR)
+if(${CMAKE_VERSION} VERSION_LESS 3.12)
+	# Use most recent policies, even for older cmake
+	cmake_policy(VERSION ${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION})
```

Comment:
The above is to specify "supported versions" and to enable all policies.
(See https://cliutils.gitlab.io/modern-cmake/chapters/basics.html)